src/core_plugins/kibana/public/visualize/wizard/step_1.html

```diff
@@ -48,6 +48,7 @@ <h2 class="kuiSubTitle kuiVerticalRhythm">
           ng-href="{{ getVisTypeUrl(type) }}"
           tooltip="{{ getVisTypeTooltip(type) }}"
           tooltip-placement="{{ getVisTypeTooltipPosition($parent.$index) }}"
+          aria-describedby="visDescription_{{::type.name}}"
```

Comment:
Good point. The name actually is kind of the `id` of the visualization and will be stored in the saved object to identify it. But honestly I am not sure what our restrictions are on this id currently, so I will check this up. Do you know by chance, where to find that function? :D